packages/account-v2/src/constants/proofofincomeConstants.ts

```diff
@@ -0,0 +1,11 @@
+import * as Yup from 'yup';
+import { proofOfIncomeUtils } from '../utils/proofOfIncomeUtils';
+
+export const listItems = [
+    'The document must be up-to-date and signed by the issuance authority.',
+    'The document must contain a letterhead.',
+    'Invalid or incomplete documents shall be rejected.',
+];
+export const documentValidation = Yup.mixed<File>();
+
+export const PoincDocumentsList = proofOfIncomeUtils();
```

Comment:
Also proofOfIncomeUtils seems to return a constant object. If  so, cant we declare it in this constant file itself?